GPU/GLES/TextureCacheGLES.cpp

```diff
@@ -471,18 +469,45 @@ void TextureCacheGLES::BuildTexture(TexCacheEntry *const entry) {
 	}
 
 	for (int i = 0; i < plan.levelsToLoad; i++) {
-		LoadTextureLevel(*entry, *plan.replaced, i == 0 ? plan.baseLevelSrc : i, i, plan.scaleFactor, dstFmt);
+		int srcLevel = i == 0 ? plan.baseLevelSrc : i;
```

Comment:
Why not `plan.baseLevelSrc + i`?

-[Unknown]

Reply:
It's to emphasize that when baseLevelSrc is used, we only load one level. Though yeah, could just add...